ci/properties/msbuild.properties.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "MSBuild based projects",
+    "description": "Build a MSBuild based project.",
+    "iconName": "c-cpp",
```

Comment:
@WillBuik is there an official icon for MSBuild (that customers will recognize)? I couldn't find anything obvious in the docs.

Reply:
Discussed offline. We'll use this icon for now, as customers likely wouldn't recognize the MSBuild icon.